### PR TITLE
fix: prompt is using the wrong group

### PR DESCRIPTION
**Pull request by 0xff-dev**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it
fix: prompt is using the wrong group

#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

File: deploy/charts/arcadia/templates/rag_job_rbac.yaml [hunk 1]
```diff
@@ -53,13 +53,6 @@ rules:
   verbs:
   - list
   - get
-- apiGroups:
```
Comment (bjwswang) [inline deploy/charts/arcadia/templates/rag_job_rbac.yaml:56]: @0xff-dev  So we do not require this permission anymore ?  Even we fix it to the correct group

Comment (0xff-dev) [reply deploy/charts/arcadia/templates/rag_job_rbac.yaml:56]: This permission is needed, I had the wrong group for the prompt resource. There's also a previous entry about prompt's permissions that fulfills the requirement, so just remove it here.

Comment (bjwswang) [reply deploy/charts/arcadia/templates/rag_job_rbac.yaml:56]: please show me the link(the previous entry)

**Review by bjwswang — APPROVED** [review]

File: deploy/charts/arcadia/templates/rag_job_rbac.yaml (hunk 1 above)
Comment (0xff-dev) [reply deploy/charts/arcadia/templates/rag_job_rbac.yaml:56]: https://github.com/kubeagi/arcadia/blob/9d3ee31434dc0e587fca1392a336fb2e945228af/deploy/charts/arcadia/templates/rag_job_rbac.yaml#L43

**Pull request merged**